core/src/klio_core/options.py

```diff
@@ -127,25 +139,127 @@ def wrapper(func):
     return wrapper
 
 
-def override(func):
+#####
+# common options
+#####
+def direct_runner(func):
     return click.option(
-        "-O",
-        "--override",
-        default=[],
-        multiple=True,
-        help="Override a config value, in the form ``key=value``.",
+        "--direct-runner",
+        default=False,
+        is_flag=True,
+        help="Run the job locally via the DirectRunner.",
     )(func)
 
 
-def template(func):
+def update(func):
     return click.option(
-        "-T",
-        "--template",
-        default=[],
-        multiple=True,
+        "--update/--no-update",
+        default=None,
+        is_flag=True,
+        help="[Experimental] Update an existing streaming Cloud Dataflow job.",
+    )(func)
+
+
+def show_logs(func):
+    return click.option(
+        "--show-logs",
+        default=False,
+        show_default=True,
+        is_flag=True,
+        help="Show a job's logs while profiling.",
+    )(func)
+
+
+#####
+# options for `klioexec profile memory
```

Comment:
The comments throughout this module are not consistent in terms of what they're for - for instance, these look specific to `klioexec` when it's also used for the klio cli. Maybe just generalize them to say "options for \`\<cmd> profile memory\`" (or whatever it's use for).